runtime/calamari/src/lib.rs

```diff
@@ -915,7 +915,8 @@ pub type Executive = frame_executive::Executive<
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
-	AllPallets,
+	AllPalletsWithSystem,
+	()
```

Comment:
Why `()` here?

Reply:
I will remove it. I think you still remember this trait `OnRuntimeUpgrade`. It's empty by default `OnRuntimeUpgrade = ()`.